webapp/packages/plugin-gis-viewer/src/GISValuePresentation.tsx

```diff
@@ -21,42 +21,53 @@ import {
 } from '@cloudbeaver/plugin-data-viewer';
 
 import { CrsInput } from './CrsInput';
+import classes from './GISValuePresentation.m.css';
 import { CrsKey, IAssociatedValue, IGeoJSONFeature, LeafletMap } from './LeafletMap';
 import { ResultSetGISAction } from './ResultSetGISAction';
 
-function getCrsKey(feature?: IGeoJSONFeature): CrsKey {
-  switch (feature?.properties.srid) {
+proj4.defs('EPSG:3395', '+title=World Mercator +proj=merc +lon_0=0 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs');
+
+function getCrsKey(srid: number): CrsKey {
+  switch (srid) {
     case 3857:
-      return 'EPSG3857';
+      return 'EPSG:3857';
```

Comment:
<img width="1121" alt="Screenshot 2024-01-30 at 11 01 40 AM" src="https://github.com/dbeaver/cloudbeaver/assets/811729/539b104b-9685-4138-b6f7-8d50b30105a1">
it's code and you can google it, i don't agree that we need to move it to const (it will be looking like `const EPSG3857 = 'EPSG:3857'`)

Reply:
moreover number we get from server and it can by typed with gql enum but it seems no sense for it (again it's not a random number it's code)